cpu/atmega_common/periph/timer.c

```diff
@@ -87,17 +88,21 @@ int timer_init(tim_t tim, unsigned long freq, timer_cb_t cb, void *arg)
     uint8_t pre = 0;
 
     /* make sure given device is valid */
-    if (tim >= TIMER_NUMOF) {
+    if (tim >= TIMER_NUMOF)
+    {
```

Comment:
Is this change the result of uncrustify ? Then I think there's an issue with the uncrustify configuration file. The opening curly brace should be on the same line as the if.